.flowconfig

```diff
@@ -2,6 +2,9 @@
 .*/node_modules/jsonlint/.*
 .*/node_modules/config-chain/test/broken.json
 
+# This is expected as this malformed file is for a test in the library.
+.*/node_modules/resolve/test/resolver/malformed_package_json/package.json
```

Comment:
Had to add this ignore here to the flowconfig as flow was throuwing an error on an external dependency which is expected as the file is part of a test.